src/plugins/directoryvalue/README.md

```diff
@@ -5,7 +5,7 @@
 - infos/provides =
 - infos/recommends =
 - infos/placements = postgetstorage presetstorage
-- infos/status = maintained nodep preview experimental unfinished concept  discouraged
+- infos/status = maintained nodep preview
```

Comment:
See also #1734

Btw. are you aware of scripts/update-infos-status?

Reply:
> Btw. are you aware of scripts/update-infos-status?

Now I am 😊.